composer.json

```diff
@@ -20,24 +20,20 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "codeception/codeception": "*@dev",
-        "codeception/lib-innerbrowser": "*@dev",
+        "codeception/codeception": "^5.0.0-RC1",
+        "codeception/lib-innerbrowser": "^3.1.1",
         "guzzlehttp/guzzle": "^7.4",
         "symfony/browser-kit": "^5.4 || ^6.0"
     },
     "require-dev": {
         "ext-curl": "*",
         "aws/aws-sdk-php": "^3.199",
-        "codeception/module-rest": "^2.0 || *@dev"
-    },
-    "conflict": {
-        "codeception/codeception": "<5.0",
-        "codeception/lib-innerbrowser": "<3.0"
+        "codeception/module-rest": "^3.1.1"
     },
     "suggest": {
         "codeception/phpbuiltinserver": "Start and stop PHP built-in web server for your tests"
     },
-    "minimum-stability": "dev",
+    "minimum-stability": "RC",
```

Comment:
`minimum-stability` must be `dev`, because we have a circular dependency here.
module-phpbrowser is a dev dependency of codeception/codeception and must be compatible with branches of codeception/codeception and in reverse, codeception/codeception must be compatible with branches of module-phpbrowser. That's why we have `*@dev` in version constraint.

Setting COMPOSER_ROOT_VERSION may help to install dependencies on a branch, but it is difficult to test.